docs/content/contributing/docs/contributing-docs/index.md

```diff
@@ -3,8 +3,7 @@ type: docs
 title: "How-To: Contribute to the Radius documentation"
 linkTitle: "Contribute to docs"
 description: "How to contribute to the Radius documentation"
-weight: 200
-slug: "docs"
+weight: 100
```

Comment:
Could you add aliases for any leaf (index.md) pages that have moved please? We want to maintain backwards compatibility with old URLs we've tweeted